codespell_lib/data/dictionary.txt

```diff
@@ -12841,6 +12841,7 @@ clame->claim
 clammer->clamber, clamor,
 clanup->cleanup
 clanups->cleanups
+clapse->collapse
```

Comment:
Could it be `claps`?

I would suggest adding all verbal forms:
```
clapsed->collapsed
clapses->collapses
clapsing->collapsing
```

Reply:
Could it also be `clasp` ?

Reply:
Hi everyone,

for me adding this, is absolutely ok:
clapsed->collapsed
clapses->collapses
clapsing->collapsing

Are you doing this or am I?

Adding 'clasp' seems to be difficult to become a false positive according to grep.app I think. What do you say?

Reply:
Please add them if possible.

How could a `clasp` **suggestion** become a false positive? On the other hand, there might be no or very few occurrences of `clapse` as a misspelling of `clasp` out there.

Reply:
> On the other hand, there might be no or very few occurrences of clapse as a misspelling of clasp out there.  

Indeed, i'm not married to the suggestion. I leave it to your discretion :saluting_face:

Reply:
1. Keys S and E are neighbours on QWERTY keyboards, so I would recommend at least:
   ```
   clapse->collapse, claps,
   ```
2. The OED has [`clapse`](https://www.oed.com/search/dictionary/?q=clapse) is a variant of `clasp`. However, it is an obsolete variant, not in use any more and missing from other mainstream dictionaries. Therefore, I would recommend all 3 suggestions:
   ```
   clapse->collapse, clasp, claps,
   ```
3. Perhaps even add `lapse` as I have found an occurrence of an OCR engine changing `lapse` into `clapse`:
   ```
   clapse->collapse, clasp, claps, lapse,
   ```

There are few occurrences of `clapse` any way:
* https://github.com/search?q=%22+clapse+%22